qiita_pet/templates/sitebase.html

```diff
@@ -145,8 +146,7 @@
                 <li><a href="/study/public/">View Public Studies</a></li>
               </ul>
             </li>
-            {% set user_tmp = handler.user() %}
-            {% set user_level = user_tmp.level if user_tmp is not None else None %}
+            {% set user_level = user.level %}
```

Comment:
Also to add to this, by making the self.current_user return a User object, tornado actually caches the user object on first instantiation, so we save all the "does the id exist?" calls made when we kept instantiating a User object in each function. This should hopefully save a lot of unneeded calls as well.